vowpalwabbit/core/src/reduction_stack.cc

```diff
@@ -250,7 +250,7 @@ void default_reduction_stack_setup::delayed_state_attach(VW::workspace& all, VW:
   _all_ptr = &all;
   _options_impl = &options;
   // populate setup_fn -> name map to be used to lookup names in setup_base
-  all.build_setupfn_name_dict(_reduction_stack);
+  all._build_setupfn_name_dict(_reduction_stack);
```

Comment:
Curious about the _ in fn_name.  What's the convention?

Reply:
These functions are implementation details, so the idea was to express that. This change is no longer needed though as it's superseded by: https://github.com/VowpalWabbit/vowpal_wabbit/pull/4456

Reply:
Got it.